includes/class-payment-information.php

```diff
@@ -264,6 +275,26 @@ public static function get_token_from_request( array $request ) {
 		return $token;
 	}
 
+	/**
+	 * Extract the payment CVC confirmation from the provided request.
+	 *
+	 * @param array $request Associative array containing payment request information.
+	 *
+	 * @return string|NULL
+	 */
+	public static function get_cvc_confirmation_from_request( array $request ) {
+		$payment_method    = $request['payment_method'] ?? null;
+		$token_request_key = 'wc-' . $payment_method . '-payment-cvc-confirmation';
```

Comment:
In the event that `$payment_method` is `null`, do we need to still generate the `$token_request_key` here? Or can we just return `null` in that case?

Reply:
This method was based on the upper method, but fixed on 8ba6fb8.